src/lsm/schema.zig

```diff
@@ -15,6 +15,7 @@
 //! Index block schema:
 //! │ vsr.Header                   │ operation=BlockType.index,
 //! │                              │ context=schema.TableIndex.Context,
```

Comment:
Unrelated: `context=schema.TableFilter.Context` as described in the "Index block" comment section is inconsistent with the lowercase `context=schema.TableFilter.context` used by the "Filter block" and "Data block" ones.

This made me wonder if `Parent` should be lowercase as well or if it's the other way around.

Maybe we could adopt `:` to indicate "type of" and `=` to indicate "value of" for those comments, for example:

```
context : schema.TableIndex.Context,
parent : schema.TableIndex.Parent,
request=@sizeOf(Key)
timestamp=snapshot_min
```